express-get-json/server.js

```diff
@@ -0,0 +1,36 @@
+import express from 'express';
+
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95,
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100,
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92,
+  },
+};
+
+app.listen(8080, () => {
```

Comment:
Although it still works the way you have it, it is customary to put the `listen` call at the very end. It makes the code more logical -- create the app, configure it, then start listening.